daq/configurator.py

```diff
@@ -164,7 +160,8 @@ def parse_args(self, args):
                 elif '=' in arg:
                     self._parse_flat_item(config, arg.split('=', 1))
                 else:
-                    self._read_config_into(config, os.getcwd(), arg)
+                    from_config_file = self.load_config(arg)
+                    config = self._deep_merge_dict(from_config_file, config)
```

Comment:
As discussed in chat, I do not think this is the right behavior. If you want to guarantee the ordering of items, then that's something that should be enforced at the higher layers of the stack (e.g. by controlling the order of args passed, not by enforcing it here).  Also, I don't think the base logic is correct since if there's multiple config files specified they will be applied in reverse order.

Reply:
Something like python configurator.py config_file1 config_file2 is not a defined behavior and is not used anywhere.

Reply:
Since DAQ only supports 1 config file, configurator should also be modified so it enforces 1 config file only.